server/init.go

```diff
@@ -39,10 +40,36 @@ type GenesisTx struct {
 }
 
 var (
-	flagOverwrite = "overwrite"
-	flagGenTxs    = "gen-txs"
-	flagIP        = "ip"
-	flagChainID   = "chain-id"
+	//--name parameter name, init gen-tx command
+	FlagName = "name"
+	//--home-client parameter name, init gen-tx command
+	FlagClientHome = "home-client"
+	//--owk parameter name, init gen-tx command
+	FlagOWK = "owk"
+
+	// bonded tokens given to genesis validators/accounts
+	FreeFermionVal = int64(100)
+	// ...
+	FreeFermionsAcc = int64(50)
+)
+
+// Storage for init command input parameters
+type InitConfig struct {
+	ChainID   string
+	GenTxs    bool
+	GenTxsDir string
+	Overwrite bool
+}
+
+var (
+	//--overwrite parameter name, init command
+	FlagOverwrite = "overwrite"
+	//--gen-txs parameter name, init command
+	FlagGenTxs = "gen-txs"
+	//--ip parameter name, init command
+	FlagIP = "ip"
+	//--chain-id parameter name, init command
+	FlagChainID = "chain-id"
```

Comment:
move the flag definitions to the same place as earlier in this file, remove comments, maybe add one single command at the top specifying that these belong to `init command`